packages/@uppy/companion/src/server/provider/drive.js

```diff
@@ -2,7 +2,8 @@ const request = require('request')
 // @ts-ignore
 const purest = require('purest')({ request })
 const logger = require('../logger')
-
+const DRIVE_FILE_FIELDS = 'kind,id,name,mimeType,ownedByMe,permissions(role,emailAddress),size,modifiedTime,iconLink,thumbnailLink'
```

Comment:
why do we need to explicitly declare these fields?

Reply:
Ah, never mind. I see the reason [here](https://developers.google.com/drive/api/v3/migration}

Reply:
The documentation also misses a critical point: not only is the default behaviour to only return "a subset of commonly used fields" - but if you _do_ specify `fields`, those commonly used fields are no longer returned unless you explicity specify them.  It's therefore necessary to explicitly request every single field which we need - not just those which aren't returned by default.